public/js/lib/editor/index.js

```diff
@@ -219,8 +221,8 @@ export default class Editor {
     makeComment.click(() => {
       utils.insertText(this.editor, '> []')
     })
+
     uploadImage.bind('change', function (e) {
-      console.log("tiggered")
```

Comment:
Good catch! Looks like someone left some surgical tools…

Reply:
Is it OK if I add `no-console` to `.eslintrc`?
Actually, now that I look at `.eslintrc`, it does look a little bit minimal. Any plans to change that?

Reply:
Yes, I think we should improve this. First step should be to get rid of the warnings (which will touch all files). If you want to go for it, feel free to go. Otherwise, I guess I'll do this before a release.